### PR TITLE
Presence Badge: Updated colour for unknown variant

### LINKED ISSUE #26876: PresenceBadge: 'Unknown' color should be the same as the 'Offline' color, i.e. '#616161'.
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
any
```

### Are you reporting Accessibility issue?

no

### Reproduction

v9 docs site

## Actual Behavior
image

## Expected Behavior
The color for [the 'Unknown' status in the Fluent UI v9 version](https://react.fluentui.dev/?path=/docs/components-badge-presencebadge--default#status) is '#d13438' (same as the 'Blocked', 'Busy', 'Busy OOF', 'Do Not Disturb' and 'Do Not Disturb OOF' colors). The 'Unknown' color should be the same as the 'Offline' color, i.e. '#616161'.

image

### Logs

_No response_

### Requested priority

High

### Products/sites affected

Teams

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

### PR DESCRIPTION
**Pull request by kkakroo**
## Previous Behavior
The color for [the 'Unknown' status in the Fluent UI v9 version](https://react.fluentui.dev/?path=/docs/components-badge-presencebadge--default#status) is '#d13438' (same as the 'Blocked', 'Busy', 'Busy OOF', 'Do Not Disturb' and 'Do Not Disturb OOF' colors).

## New Behavior
The 'Unknown' color should be the same as the 'Offline' color, i.e. '#616161'.

## Related Issue(s)

- Fixes #26876

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                            |     Baseline (minified/GZIP) |                           PR |                                                                                                                                                                                                  Change |
| :----------------------------------------------------------------------------------------------------------- | ---------------------------: | ---------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                      | `87.465 kB`<br />`21.569 kB` | `87.614 kB`<br />`21.584 kB` | `149 B` <br />`15 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                   | `51.192 kB`<br />`14.043 kB` | `51.341 kB`<br />`14.059 kB` | `149 B` <br />`16 B`  |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr> | `67.414 kB`<br />`18.551 kB` | `67.563 kB`<br />`18.568 kB` | `149 B` <br />`17 B`  |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>      |  `25.438 kB`<br />`7.254 kB` |  `25.587 kB`<br />`7.271 kB` | `149 B` <br />`17 B`  |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                | `58.207 kB`<br />`15.986 kB` | `58.356 kB`<br />`16.006 kB` | `149 B` <br />`20 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |    `15.42 kB`<br />`6.167 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `23.113 kB`<br />`7.069 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `24.002 kB`<br />`7.374 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |   `64.09 kB`<br />`17.544 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> |  `201.998 kB`<br />`56.27 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `35.049 kB`<br />`11.533 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.324 kB`<br />`2.129 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `143.646 kB`<br />`39.157 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `43.776 kB`<br />`12.063 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 |   `134.4 kB`<br />`34.514 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |  `82.316 kB`<br />`20.576 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |  `81.637 kB`<br />`20.386 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/30ad74d01e550256a7f62dd8760d393d54473959'>30ad74d01e550256a7f62dd8760d393d54473959</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Avatar-mount_base.html">1317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Avatar-mount.html">1308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Button-mount_base.html">942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Button-mount.html">938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Field-mount_base.html">2061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Field-mount.html">2043</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProvider-mount_base.html">1615</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProvider-mount.html">1632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProviderWithTheme-mount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProviderWithTheme-mount.html">626</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">585</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">637</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/InfoButton-mount_base.html">563</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/InfoButton-mount.html">562</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/MakeStyles-mount_base.html">1915</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/MakeStyles-mount.html">1933</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Persona-mount_base.html">3091</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/Persona-mount.html">3046</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/SpinButton-mount_base.html">2453</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26970/perf-test-react-components/SpinButton-mount.html">2479</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by behowell** [comment]
Hi @kkakroo, thanks for the PR! The PresenceBadge's unknown state (red) is by design. I'm going to double-check with our designers to see if it should be updated to be gray instead.

**Review by behowell — COMMENTED** [review]
I've confirmed that the design should be updated, thanks.

It looks like you'll need to add a change file. Run `yarn change` from the root of your enlistment and it'll prompt you for a change description.

File: packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts [hunk 1]
```diff
@@ -101,12 +107,14 @@ export const usePresenceBadgeStyles_unstable = (state: PresenceBadgeState): Pres
...
     state.status === 'offline' && styles.statusOffline,
     state.status === 'out-of-office' && styles.statusOutOfOffice,
+    state.status === 'unknown' && styles.statusUnknown,
     state.outOfOffice && styles.outOfOffice,
     state.outOfOffice && state.status === 'available' && styles.outOfOfficeAvailable,
     state.outOfOffice && isBusy && styles.outOfOfficeBusy,
     state.outOfOffice && state.status === 'away' && styles.statusOutOfOffice,
     state.outOfOffice && state.status === 'offline' && styles.statusOffline,
     state.outOfOffice && state.status === 'out-of-office' && styles.statusOutOfOffice,
+    state.outOfOffice && state.status === 'unknown' && styles.statusUnknown,
```
[the remark this answers is not in the document]
Comment (kkakroo) [reply packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts]: thanks for pointing it out, updated.

**Comment by kkakroo** [comment]
Hi @behowell, updates made as per review comments.

**Review by behowell — APPROVED** [review]
Looks good, thanks!

**Review by khmakoto — APPROVED** [review]

**Comment by kkakroo** [comment]
Hi,is there something pending from my side on this?
@behowell @khmakoto

**Comment by behowell** [comment]
Hi @kkakroo, no there's nothing left for you to do. There seems to be a problem with the build pipeline for this PR. We're investigating and will merge once it's figured out. Thanks!

**Pull request merged**